public/css/list/item-list.less

```diff
@@ -170,3 +170,17 @@
     position: relative;
   }
 }
+
+.controls .list-item:not(:only-child):not(:last-child) {
```

Comment:
You can remove the `:not(:only-child)` selector. If it's the only child, it's also the last.

Reply:
Smart. 💡

Reply:
✅